appveyor.yml

```diff
@@ -1,37 +1,49 @@
 # Based on the "trust" template v0.1.1
 # https://github.com/japaric/trust/tree/v0.1.1
 
+os: Visual Studio 2015
+
 environment:
   global:
-    RUST_VERSION: stable
     CRATE_NAME: cobalt
     APPVEYOR_CACHE_SKIP_RESTORE: true
 
   matrix:
     # Stable channel
-    - TARGET: x86_64-pc-windows-gnu
-      CHANNEL: stable
-    - TARGET: i686-pc-windows-msvc
-      CHANNEL: stable
+    #- TARGET: x86_64-pc-windows-gnu
+    #  CHANNEL: stable
```

Comment:
If we're not supporting gnu, we should remove it (Rather than leave it commented out) and give a reason in the commit message.